sysutils/fastfetch/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+PortGroup        github 1.0
+PortGroup        cmake 1.0
+
+github.setup     fastfetch-cli fastfetch 2.5.0
+platforms        any
```

Comment:
does thus install the same files on each OS version? I kind of doubt that since it's compiling stuff, so perhaps just remove the line and keep it the default (i.e., "darwin").